ansible/inventories/staging/group_vars/all/vars.yml

```diff
@@ -113,6 +113,7 @@ inventory_ckan_solr_port: "8983"
 nrinfragent_license_key: "{{ vault_nrinfragent_license_key }}"
 
 
+pycsw_app_environment: staging
```

Comment:
Is this used anywhere? I don't see it in the other vars.yml files

Reply:
It is undocumented, and somewhat of a hack for [gsa/datagov-deploy-pycsw](https://github.com/GSA/datagov-deploy-pycsw/search?q=pycsw_app_environment&unscoped_q=pycsw_app_environment).